src/aics-image-viewer/components/App/types.ts

```diff
@@ -29,16 +29,26 @@ type ControlNames =
 /** Show/hide different elements of the UI */
 export type ControlVisibilityFlags = { [K in ControlNames]: boolean };
 
+export type MultisceneUrls = { scenes: (string | string[])[] };
```

Comment:
why not
```
{ 
  scenes: 
  {
    sources :string[]
  } []
}
```
 ?

Reply:
or to be even more explicit and readable, something like:
export type Scene = { sources: string[] };
export type MultiSceneUrls = Scene[];

I'm still not sure why you need to have `string | string[]`

Reply:
My thought here was mainly backwards compatibility... though I suppose the multi-source images we have out in the wild are expressed as URLs and so we can just choose to parse them into a new props type?

Reply:
that was my thought, use the url parsing code to transform them into the props type that we want.  Maybe it's ok that the props are still general to string or array of string or multiscene struct, but as early as possible it should all get converted into a multiscene struct for the rest of the app to use.